Tutorials/Python_Developer_Tutorials/foundation/Commanding-Local/Proxy-Local-Commands-Tutorial_.py

```diff
@@ -1,87 +1,110 @@
 # This example will show how to command Faraday using the basic command application.
-# LED's being commanded may be recieving other commands and not work as intended (i.e. RED due to RF TX indication)
+# LED's being commanded may be receiving other commands and not work as intended (i.e. RED due to RF TX indication)
 
-#Imports - General
+# Imports - General
 
-import os, sys
-sys.path.append(os.path.join(os.path.dirname(__file__), "../../../../Faraday_Proxy_Tools")) #Append path to common tutorial FaradayIO module
+import os
+import sys
+import time
+import ConfigParser
+
+sys.path.append(os.path.join(os.path.dirname(__file__),
+                             "../../../../Faraday_Proxy_Tools"))  # Append path to common tutorial FaradayIO module
 
-#Imports - Faraday Specific
+# Imports - Faraday Specific
 from FaradayIO import faradaybasicproxyio
 from FaradayIO import faradaycommands
 from FaradayIO import gpioallocations
 
-import time
-
-#Definitions
+# Open configuration INI
+config = ConfigParser.RawConfigParser()
+filename = os.path.abspath("configuration.ini")
+config.read(filename)
 
+# Definitions
 
-#Variables
-local_device_callsign = 'REPLACEME'  # Should match the connected Faraday unit as assigned in Proxy configuration
-local_device_node_id = REPLACEME  # Should match the connected Faraday unit as assigned in Proxy configuration
+# Variables
+local_device_callsign = config.get("DEVICES",
+                                   "UNIT0CALL")  # Should match the connected Faraday unit as assigned in Proxy configuration
+local_device_node_id = config.get("DEVICES",
+                                  "UNIT0ID")  # Should match the connected Faraday unit as assigned in Proxy configuration
+local_device_callsign = str(local_device_callsign).upper()
 
-#Start the proxy server after configuring the configuration file correctly
-#Setup a Faraday IO object
-faraday_1 = faradaybasicproxyio.proxyio() #default proxy port
+# Start the proxy server after configuring the configuration file correctly
+# Setup a Faraday IO object
+faraday_1 = faradaybasicproxyio.proxyio()  # default proxy port
 faraday_cmd = faradaycommands.faraday_commands()
 
-
 ##############
 ## TOGGLE GPIO
 ##############
-#Turn LED 1 ON LOCAL
+# Turn LED 1 ON LOCAL
 print "Turning ON the Green LED (LED #1)"
```

Comment:
Well I agree our applications and larger tutorials should be upgraded to logging this is simply actively telling the user when the LED should be ON. Print is appropriate.